django/chat/models.py

```diff
@@ -515,6 +515,8 @@ class AnswerSource(models.Model):
     min_page = models.IntegerField(null=True)
     max_page = models.IntegerField(null=True)
 
+    claims_list = models.JSONField(default=list, blank=True)
```

Comment:
Add nullable "highlighted_text" as a property here also, then you only have to highlight it once (subsequent clicks of 'sources' will not run it again). Update the `node_text` property so that it will use highlighted_text if it exists.